docs/blog/2025-01-21-project-references.md

```diff
@@ -0,0 +1,177 @@
+---
+title: What Are TypeScript Project References?
+slug: typescript-project-references
+authors: [Zack DeRose]
+tags: [typescript, monorepo, nx]
+cover_image: /blog/images/2025-01-21/ts-islands.png
+---
+
+## Connecting Projects At The TypeScript Level
```

Comment:
Having a title right in the beginning always feels off to me as there's already the main article title immediately followed by another one :) I think we can just drop it here and this first paragraphs serve as the intro

Reply:
I agree with Juri that the title underneath the graphic feels off. Maybe one intro sentence makes more sense. Like: `Project References is a feature to help scale your TypeScript monorepo. In this post you will learn all about how it works and how it helps with the performance of your workspaces.